googleapiclient/errors.py

```diff
@@ -126,7 +126,10 @@ def __init__(self, reason, resp=None, content=None):
     self.reason = reason
 
   def __repr__(self):
+    if getattr(self.resp, 'status', None) is not None:
```

Comment:
I'm suggesting that you change
```
if <expression> is not None:
  return <A>
else:
  return <B>
```
to
```
if <expression> is None:
  return <B>
else:
  return <A>
```
. Doesn't that have (one syntax element's worth) less complexity with identical behavior?